index.js

```diff
@@ -43,21 +43,6 @@ module.exports = {
     }
   },
 
-  shouldIncludeChildAddon: function(addon) {
```

Comment:
Add this back in, but make the check for just glimmer.

Add a note to README that if they're using 2.3.0-beta +that they should add `ember-getowner-polyfill` to their blacklist in `ember-cli-build.js` (`this.options.addons.blacklist = ['ember-getowner-polyfill']`).